beacon-chain/core/blocks/proposer_slashing.go

```diff
@@ -13,7 +13,7 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-type slashValidatorFunc func(state.BeaconState, types.ValidatorIndex) (state.BeaconState, error)
```

Comment:
maybe add in named parameters to the type ? Any future user will understand what the last 2 values are for.Otherwise its just `uint64,uint64`